<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralOverview</class>
 <widget class="QDialog" name="GeneralOverview">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>General Overview</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>441</width>
     <height>461</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <column>
    <property name="text">
     <string>General</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Kingdom</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Gender</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>MaxHP</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Package</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>450</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="generalPhoto">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>10</y>
     <width>200</width>
     <height>290</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../generals/card/caocao.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="skillLabel">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>310</y>
     <width>30</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Skill</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="skillTextEdit">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>330</y>
     <width>201</width>
     <height>111</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>GeneralOverview</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>551</x>
     <y>492</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>492</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
